From 1571e6da8a90bb325a94330dcd130d56bae30b37 Mon Sep 17 00:00:00 2001
From: Roy Li <rongqing.li@windriver.com>
Date: Thu, 20 Feb 2014 17:07:05 +0800
Subject: [PATCH] policy/modules/services/crontab: allow sysadm_r to run
 crontab

This permission has been given if release is not redhat; but we want it
even we define distro_redhat

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Roy Li <rongqing.li@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/roles/sysadm.te | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/policy/modules/roles/sysadm.te b/policy/modules/roles/sysadm.te
index 1de7e441d..129e94229 100644
--- a/policy/modules/roles/sysadm.te
+++ b/policy/modules/roles/sysadm.te
@@ -1277,6 +1277,10 @@ optional_policy(`
 	zebra_admin(sysadm_t, sysadm_r)
 ')
 
+optional_policy(`
+	cron_admin_role(sysadm_r, sysadm_t)
+')
+
 ifndef(`distro_redhat',`
 	optional_policy(`
 		auth_role(sysadm_r, sysadm_t)
@@ -1295,10 +1299,6 @@ ifndef(`distro_redhat',`
 		chromium_role(sysadm_r, sysadm_t)
 	')
 
-	optional_policy(`
-		cron_admin_role(sysadm_r, sysadm_t)
-	')
-
 	optional_policy(`
 		cryfs_role(sysadm_r, sysadm_t)
 	')
-- 
2.17.1

